Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -624,6 +624,7 @@ private DeployResult GetResult(string id, string activeDeploymentId, bool isDepl
                 {
                     NextManifestFilePath = GetDeploymentManifestPath(id),
                     PreviousManifestFilePath = GetActiveDeploymentManifestPath(),
+                    IgnoreManifest = deploymentInfo.CleanupTargetDirectory,
```

Comment:
It's not instantly clear to me how ignoring the manifest results in cleaning up the target folder. A comment would be useful here.

Reply:
Done.

Reply:
Thanks, I guess I had misunderstood/forgotten when `IgnoreManifest` meant, but now I understand.